### PR TITLE
Update businesses.json

### PR DESCRIPTION
**Pull request by lyonjinx**
updated business cards w/ info from slide 37

### CODE REVIEW

File: src/data/businesses.json [hunk 1]
```diff
@@ -28,33 +28,38 @@
   "medFindIt": {
     "id": "medFindIt",
     "name": "Med Find It",
-    "ownerId": null,
+    "ownerId": "Greg Malonas",
```
Comment (markalanrichards) [inline src/data/businesses.json:31]: ownerIds need to match ids in https://github.com/markalanrichards/ownemployed/blob/master/src/data/users.json

File: src/data/businesses.json [hunk 2]
```diff
@@ -28,33 +28,38 @@
...
     "tagline": "A social platform to find innovative medical solutions.",
     "description": "",
-    "status": "open"
+    "status": "Defining value proposition",
+    "Contributors":"3"
+"
```
Comment (markalanrichards) [inline src/data/businesses.json:36]: I think this `"` is incompatible with Json; hence the errors.

File: src/data/businesses.json (hunk 1 above)
Comment (gwens) [reply src/data/businesses.json:31]: @lyonjinx to add a bunch of users to the database quickly, the minimum data they need is an ID and a name, and the relevant businessIds that they are connected to. It's ok if the rest is missing, e.g.

```
{
  "gregMalonas": {
    "id": "gregMalonas",
    "name: "Greg Malonas",
    "ownedModels": ["medFindIt"]
  }
}
```

Comment (lyonjinx) [reply src/data/businesses.json:31]: done. sent for review

**Pull request closed without merging**